tests/foreman/ui/test_user.py

```diff
@@ -1168,3 +1056,57 @@ def test_positive_mails_for_new_timezone(self):
 
         @CaseLevel: Integration
         """
+
+
+class ActiveDirectoryUserTestCase(UITestCase):
```

Comment:
Thought about moving that case into `test_adusergroup` module as it has same `setup` class, but logically it is absolutely wrong, so decided create own class for user and LDAP